defaults.lua

```diff
@@ -251,6 +251,10 @@ NETWORK_PROXY = nil
 -- Use turbo library to handle async HTTP request
 DUSE_TURBO_LIB = false
 
+-- Absolute path to stardict files.
+-- By default they're stored in data/dict under dataDir.
+STARDICT_DATA_DIR = nil
```

Comment:
We should consider something like this to keep compatibility with system-wide sdcv configuration out of the box. Unless sdcv picks up on that regardless?
```suggestion
STARDICT_DATA_DIR = os.getenv("STARDICT_DATA_DIR")
```

Reply:
I don't know:

```bash
pazos@bowie:/k$ ./sdcv --only-data-dir -l
Dictionary's name   Word count
pazos@bowie:/k$ export STARDICT_DATA_DIR=/test/dict/
pazos@bowie:/k$ ./sdcv --only-data-dir -l
Dictionary's name   Word count
pazos@bowie:/k$ ./sdcv --only-data-dir --data-dir=/test/dict -l
Dictionary's name   Word count
Oxford English Dictionary    75753
Diccionario Español RAE    86023
Diccionario Inglés-Español    20737
```

Reply:
--only-data-dir overrides it

https://github.com/Dushistov/sdcv/blob/08461acab8e25dc3b1a596fc7146e079dcef3d64/src/sdcv.cpp#L131-L142

Reply:
Sadly it's not possible to commit your suggestion as it clashes with advanced settings. One way to circumvent this is

```lua
--- a/frontend/apps/reader/modules/readerdictionary.lua
+++ b/frontend/apps/reader/modules/readerdictionary.lua
@@ -101,7 +101,8 @@ end
 
 function ReaderDictionary:init()
     self.ui.menu:registerToMainMenu(self)
-    self.data_dir = os.getenv("STARDICT_DATA_DIR") or
+    self.data_dir = STARDICT_DATA_DIR or
+        os.getenv("STARDICT_DATA_DIR") or
         DataStorage:getDataDir() .. "/data/dict"
```

and leave `STARDICT_DATA_DIR = nil`

Reply:
That was my first thought actually, but I figured it might be slightly more elegant in defaults.lua.

But I suppose I was mixing logic and configuration, actually being inelegant.

tl;dr Yes please! :+1:

Reply:
done!